### ISSUE TITLE
Profile "four oh four"

### ISSUE THREAD

**Issue by john-light** [gitcoinco/web#2254]
**Describe the bug**
When I click on my profile I get a "four oh four" error.

**To Reproduce**
- Log in with GitHub
- Click profile
- See error

**Expected behavior**
I expect to be able to see my profile without an error

**Screenshots**
![gitcoinfourohfour](https://user-images.githubusercontent.com/9424721/45794736-f2b0a780-bc4c-11e8-807e-deda587dc2b0.png)

**Desktop (please complete the following information):**

- OS: Linux Mint 18.3
- Browser Firefox 
- Browser Version 62.0

**Comment by owocki** [comment]
@john-light can you go to https://gitcoin.co/settings/privacy

and uncheck ' Hide my profile from the leaderboard '

and click 'Save'? 

that should do the trick!

Fix it twice suggestion for the product team at Gitcoin: @SaptakS @thelostone-mc @PixelantDesign  -- if you are viewing your own profile and it's hidden, tell the user how to manage the privacy / visibility of their profile

**Comment by SaptakS** [comment]
@owocki makes sense. We can maybe also show the user their own profile but not show to others?

**Comment by john-light** [comment]
@owocki I prefer to stay off the leaderboard ;) +1 to @SaptakS suggestion to show users their own profile but not show others with the hidden option selected

**Comment by owocki** [comment]
> @owocki makes sense. We can maybe also show the user their own profile but not show to others?

sure, but i think this becomes confusing because you might assume that since you can see your profile, that others can too.. perhaps we add a little red warning up top that warns the user of this situation IFF They visit their profile under this condition?

**Comment by john-light** [comment]
@mbeacom that does seem to be the case. I can see on /profile but not /profile/john-light. You can re-open if you'd like to still track this issue.

**Comment by john-light** [comment]
@mbeacom that does seem to be the case. I can see on /profile but not /profile/john-light. You can re-open if you'd like to still track this issue.
